geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -398,6 +400,17 @@ See <a href="../../../configuring/cluster_config/gfsh_persist.html">Overview of
 <td>cluster-config</td>
 </tr>
 <tr class="even">
+<td><span class="keyword parmname">\-\-http-service-port</span></td>
+<td>Specifies the port on which the HTTP service will listen.</td>
```

Comment:
Maybe this should be "Specifies the port on which the HTTP service is started on